operator/v1/types.go

```diff
@@ -45,6 +46,22 @@ type ClusterManagerSpec struct {
 	// NodePlacement enables explicit control over the scheduling of the deployed pods.
 	// +optional
 	NodePlacement NodePlacement `json:"nodePlacement,omitempty"`
+
+	// DeployOption contains the options of deploying a cluster-manager
+	// +optional
```

Comment:
Default mode is used if DeployOption is not set.

Reply:
Add the comments

Reply:
Added.